src/common/services/block.service.ts

```diff
@@ -80,13 +80,14 @@ export default class BlockService {
 
       if (
         tx_result?.found &&
-        tx_result.result.tx_type === 'contract_call' &&
+        ['contract_call', 'smart_contract'].includes(tx_result.result.tx_type) &&
         tx_result.result.tx_status === 'success'
       ) {
-        const tx: ContractCallTransaction = tx_result?.result;
+        const tx: any = tx_result?.result;
```

Comment:
I suggest to use ContractCallTransaction | SmartContractTransaction instead of any to maintain typing support